resources/aws/environmentDefaults.go

```diff
@@ -75,13 +76,14 @@ func sandboxDefault() environmentDefault {
 			region: string(aws.RegionUSEast1),
 		},
 		ddInfra: ddInfra{
-			defaultVPCID:               "vpc-d1aac1a8",
-			defaultSubnets:             []string{"subnet-b89e00e2", "subnet-8ee8b1c6", "subnet-3f5db45b"},
-			defaultSecurityGroups:      []string{"sg-46506837", "sg-7fedd80a", "sg-0e952e295ab41e748"},
-			defaultInstanceType:        "t3.medium",
-			defaultARMInstanceType:     "t4g.medium",
-			defaultInstanceStorageSize: 200,
-			defaultShutdownBehavior:    "stop",
+			defaultVPCID:                 "vpc-d1aac1a8",
+			defaultSubnets:               []string{"subnet-b89e00e2", "subnet-8ee8b1c6", "subnet-3f5db45b"},
+			defaultSecurityGroups:        []string{"sg-46506837", "sg-7fedd80a", "sg-0e952e295ab41e748"},
+			defaultInstanceType:          "t3.medium",
+			defaultDockerInstanceProfile: "agent-ecr-read",
```

Comment:
As I did not see a PR on `infra-cleaner`, I assume it's deleted already.
Also I don't think we need a role per use case. It could be better to have a default `vmInstanceRole` as we have a default `ecsInstanceRole` below.

Reply:
If we are fine with VM outside of Docker scenario created with this role, I'll change it